src/components/Reservations/MyReservations.js

```diff
@@ -1,8 +1,45 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import style from './MyReservation.module.css';
+import { convertDateWithName } from '../../helpers/format/format';
+
+import getReservations from '../../redux/reservation/reservationThunks';
 
 function MyReservations() {
+  const dispatch = useDispatch();
+  const reservation = useSelector((state) => state.reservationReducer);
+  useEffect(() => {
+    dispatch(getReservations());
+  }, []);
+
   return (
-    <div>MyReservations</div>
+    <div>
+      <table className={style.table} id="table">
+        <thead>
+          <tr>
```

Comment:
It would be nice to remove the border in the table and also reduce the size of the image in the mobile view, this will enhance the look on the mobile devise